types/ckeditor__ckeditor5-alignment/ckeditor__ckeditor5-alignment-tests.ts

```diff
@@ -1,12 +1,12 @@
-import Alignment from "@ckeditor/ckeditor5-alignment/src/alignment";
+import { Alignment } from "@ckeditor/ckeditor5-alignment";
 import AlignmentCommand from "@ckeditor/ckeditor5-alignment/src/alignmentcommand";
 import * as utils from "@ckeditor/ckeditor5-alignment/src/utils";
 import { Editor } from "@ckeditor/ckeditor5-core";
 import { Locale } from "@ckeditor/ckeditor5-utils";
 
 class MyEditor extends Editor {}
 
-let bool = true;
+const bool = true;
```

Comment:
Maybe `/// $ExpectType boolean` maybe?

Reply:
I didn't know about that trick. I've updated the PR.